src/Uno.UI.Toolkit/UIElementExtensions.cs

```diff
@@ -58,5 +59,129 @@ private static void OnElevationChanged(DependencyObject dependencyObject, Depend
 		}
 
 		#endregion
+
+#if NETFX_CORE
+		public static Thickness GetPadding(this UIElement uiElement)
+		{
+			if (uiElement is FrameworkElement fe && fe.TryGetPadding(out var padding))
+			{
+				return padding;
+			}
+
+			var property = uiElement.GetDependencyPropertyUsingReflection("PaddingProperty");
+			return property == null ? default : (Thickness)uiElement.GetValue(property);
+		}
+
+		public static bool SetPadding(this UIElement uiElement, Thickness padding)
+		{
+			if (uiElement is FrameworkElement fe && fe.TrySetPadding(padding))
+			{
+				return true;
+			}
+
+			var property = uiElement.GetDependencyPropertyUsingReflection("PaddingProperty");
+			if (property != null)
+			{
+				uiElement.SetValue(property, padding);
```

Comment:
If we push a value in the DP like this, we should make sure the DP is of the right type

Reply:
Fixed.